### PR TITLE
qemu.test: fix guest didn't use kvm-clock in migration_multi_host_timedrift

### PR DESCRIPTION
**Pull request by XueqiangWei**
1. Delete redundant characters in return value

Signed-off-by: Xueqiang Wei <xuwei@redhat.com>
ID: 1411686

### CODE REVIEW

**Comment by XueqiangWei** [comment]
10:03:35 INFO | 			GOOD	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.with_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.with_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	timestamp=1484042617	localtime=Jan 10 10:03:37	completed successfully
10:03:35 INFO | 		END GOOD	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.with_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.with_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	timestamp=1484042617	localtime=Jan 10 10:03:37	
10:03:35 INFO | 	END GOOD	----	----	timestamp=1484042617	localtime=Jan 10 10:03:37	
10:03:35 INFO | 			GOOD	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.with_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.with_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	timestamp=1484042618	localtime=Jan 10 10:03:38	completed successfully
10:03:35 INFO | 		END GOOD	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.with_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.with_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	timestamp=1484042618	localtime=Jan 10 10:03:38	
10:03:36 INFO | 	END GOOD	----	----	timestamp=1484042618	localtime=Jan 10 10:03:38
10:12:13 INFO | 			GOOD	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.without_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.without_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	timestamp=1484043136	localtime=Jan 10 10:12:16	completed successfully
10:12:13 INFO | 		END GOOD	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.without_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.without_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	timestamp=1484043136	localtime=Jan 10 10:12:16	
10:12:14 INFO | 	END GOOD	----	----	timestamp=1484043136	localtime=Jan 10 10:12:16	
10:12:16 INFO | Client complete
10:12:24 INFO | 			GOOD	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.without_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.without_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	timestamp=1484043146	localtime=Jan 10 10:12:26	completed successfully
10:12:24 INFO | 		END GOOD	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.without_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	virt.case2.remote_nfs.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.multi_host.migrate_multi_host.timedrift.without_kvmclock.vhost_src_yes_dst_yes.mig_online.tcp	timestamp=1484043146	localtime=Jan 10 10:12:26	
10:12:24 INFO | 	END GOOD	----	----	timestamp=1484043146	localtime=Jan 10 10:12:26

File: multi_host_migration/tests/migration_multi_host_timedrift.py [hunk 1]
```diff
@@ -69,6 +70,8 @@ def check_diff(self, mig_data):
                     check_clocksource_cmd = params.get("check_clocksource_cmd")
                     clocksource = params.get("clocksource", "kvm-clock")
                     current_clocksource = session.cmd(check_clocksource_cmd)
+                    current_clocksource = re.findall(clocksource, current_clocksource)
+                    current_clocksource = "".join(current_clocksource)
```
Comment (suqinhuang) [inline multi_host_migration/tests/migration_multi_host_timedrift.py]: the output is "kvm-clock",why need these two lines

Comment (XueqiangWei) [inline multi_host_migration/tests/migration_multi_host_timedrift.py]: the output is not "kvm-clock", it contains redundant characters (such as "crtl"), need to remove them

**Comment by CongLi** [comment]
LGTM, 

Acked-by: Cong Li <coli@redhat.com>

**Comment by suqinhuang** [comment]
ack

**Pull request merged**